docs/gallery/interactive/sofar_introduction.ipynb

```diff
@@ -0,0 +1,369 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating SOFA objects\n",
+    "\n",
+    "To cover a variety of data, SOFA offers different *conventions*. A convention defines, what data can be saved and how it is saved. You should always find the most specific convention for your data. This will help you to identify relevant data and meta data that you should provide along the actual acoustic data. Using sofar, a list of possible conventions can be obtained with"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import sofar as sf\n",
+    "import os\n",
+    "\n",
+    "sf.list_conventions()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let us assume, that you want to store head-related impulse responses (HRIRs). In this case the most specific convention is *SimpleFreeFieldHRIR*. To create a SOFA object use"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sofa = sf.Sofa('SimpleFreeFieldHRIR')\n",
+    "print(sofa)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The return value `sofa` is a `sofar.Sofa` object filled with the default values of the `SimpleFreeFieldHRIR` convention. Note that you can also get a sofa object that has only the mandatory attributes by calling `sf.Sofa('SimpleFreeFieldHRIR', mandatory=True)`. However, it is recommended to start with all attributes and discard empty optional attributes before saving the data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Getting information about SOFA objects\n",
+    "\n",
+    "An formal description of each convention is given in the [sofar documentation](https://sofar.readthedocs.io/en/stable/resources/conventions.html). You might have noted from the documentation that three different kinds of data types can be stored in SOFA files:\n",
+    "\n",
+    "* **Attributes:**\n",
+    "    Attributes are meta data stored as strings. There are two kinds of attributes. Global attributes give information about the entire data stored in a SOFA file. All entires starting with *GLOBAL* are such attributes. Specific attributes hold meta data for a certain variable. These attributes thus start with the name of the variable followed by an underscore, e.g., *ListenerPosition_Units*. An exception to this rule are the data variables, e.g, *Data_IR* is not an attribute but a double variable.\n",
+    "* **Double Variables:**\n",
+    "    Variables of type *double* store numeric data and can be entered as numbers, lists, or numpy arrays.\n",
+    "* **String Variables:**\n",
+    "    Variables of type *string* store strings and can be entered as strings, lists of string, or numpy string arrays.\n",
```

Comment:
they are strings and can be enterd as arrays? is this right?

Reply:
yes, arrays of strings